catalogue_graph/src/transformers/catalogue/concepts_transformer.py

```diff
@@ -1,21 +1,28 @@
 from collections.abc import Generator
 
+import config
 from models.graph_edge import ConceptHasSourceConcept, ConceptHasSourceConceptAttributes
 from models.graph_node import Concept
 from sources.catalogue.concepts_source import CatalogueConceptsSource
 from transformers.base_transformer import BaseTransformer
+from utils.elasticsearch import get_standard_index_name
 from utils.types import WorkConceptKey
 
 from .id_label_checker import IdLabelChecker
 from .raw_concept import RawCatalogueConcept
+from .works_transformer import ES_FIELDS, ES_QUERY
 
 
 class CatalogueConceptsTransformer(BaseTransformer):
-    def __init__(self, url: str):
-        self.source = CatalogueConceptsSource(url)
-        self.id_label_checker = IdLabelChecker(
-            node_types=["concepts", "locations", "names"], sources=["loc", "mesh"]
+    def __init__(self, pipeline_date: str | None, is_local: bool):
+        index_name = get_standard_index_name(
```

Comment:
I wonder if we could move `get_standard_index_name` inside `ElasticsearchSource` so that `pipeline_date` doesn't need to be referenced here and also when handed to descendants of ``ElasticsearchSource`?